src/main/java/com/example/ludogoriesoft/lukeriaerpapi/dtos/ProductDTO.java

```diff
@@ -20,4 +20,12 @@ public class ProductDTO {
     private int availableQuantity;
     @Size(min = 2, max = 10, message = "Кода на продукта трябда да бъде между 2 и 10 символа!")
     private String productCode;
+    public ProductDTO(BigDecimal price, Long packageId, int availableQuantity, String productCode) {
+        this.price = price;
+        this.packageId = packageId;
+        this.availableQuantity = availableQuantity;
+        this.productCode = productCode;
+    }
 }
```

Comment:
we have AllArgsConstructor annotation and I think no constructors need to be defined

Reply:
the AllArgsConstructor requires id as a param, this one doesn't.